KomaMRIFiles/src/KomaMRIFiles.jl

```diff
@@ -1,7 +1,7 @@
 module KomaMRIFiles
 
 using KomaMRIBase
-using Scanf, FileIO, HDF5, MAT    # IO related
+using Scanf, FileIO, HDF5, MAT , NIfTI   # IO related
```

Comment:
Remove NIfTI functions. That should be in another PR.

Reply:
Resolved in [3772a59](https://github.com/JuliaHealth/KomaMRI.jl/pull/184/commits/3772a5931531129a53ee51c6c0ae3e179bbdb954)